### PR TITLE
Linkify reference to EE on details page

### PR DESCRIPTION
**Pull request by nixocio**
Linkify reference to EE on a few details page.

See: https://github.com/ansible/awx/issues/9189

### CODE REVIEW

File: awx/ui_next/src/screens/Organization/OrganizationDetail/OrganizationDetail.jsx [hunk 1]
```diff
@@ -97,7 +97,13 @@ function OrganizationDetail({ i18n, organization }) {
         {summary_fields?.default_environment?.name && (
           <Detail
             label={i18n._(t`Default Execution Environment`)}
-            value={summary_fields.default_environment.name}
+            value={
+              <Link
+                to={`/execution_environments/${summary_fields.default_environment.id}/details`}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Organization/OrganizationDetail/OrganizationDetail.jsx:102]:
```suggestion
                to={`/execution_environments/${summary_fields?.default_environment?.id}/details`}
```

Good rule of thumb: Don't trust summary fields

File: awx/ui_next/src/screens/Organization/OrganizationDetail/OrganizationDetail.jsx [hunk 2]
```diff
@@ -97,7 +97,13 @@ function OrganizationDetail({ i18n, organization }) {
...
+              >
+                {summary_fields.default_environment.name}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Organization/OrganizationDetail/OrganizationDetail.jsx:104]:
```suggestion
                {summary_fields?.default_environment?.name}
```

File: awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx [hunk 3]
```diff
@@ -128,7 +128,13 @@ function WorkflowJobTemplateDetail({ template, i18n }) {
         {summary_fields?.execution_environment && (
           <Detail
             label={i18n._(t`Execution Environment`)}
-            value={summary_fields.execution_environment.name}
+            value={
+              <Link
+                to={`/execution_environments/${summary_fields.execution_environment.id}/details`}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx:133]:
```suggestion
                to={`/execution_environments/${summary_fields?.execution_environment?.id}/details`}
```

File: awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx [hunk 4]
```diff
@@ -128,7 +128,13 @@ function WorkflowJobTemplateDetail({ template, i18n }) {
...
+              >
+                {summary_fields.execution_environment.name}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx:135]:
```suggestion
                {summary_fields?.execution_environment?.name}
```

File: awx/ui_next/src/screens/Template/JobTemplateDetail/JobTemplateDetail.jsx [hunk 5]
```diff
@@ -209,7 +209,13 @@ function JobTemplateDetail({ i18n, template }) {
         {summary_fields?.execution_environment && (
           <Detail
             label={i18n._(t`Execution Environment`)}
-            value={summary_fields.execution_environment.name}
+            value={
+              <Link
+                to={`/execution_environments/${summary_fields.execution_environment.id}/details`}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Template/JobTemplateDetail/JobTemplateDetail.jsx:214]:
```suggestion
                to={`/execution_environments/${summary_fields?.execution_environment?.id}/details`}
```

File: awx/ui_next/src/screens/Project/ProjectDetail/ProjectDetail.jsx [hunk 6]
```diff
@@ -132,10 +132,15 @@ function ProjectDetail({ project, i18n }) {
         {summary_fields?.default_environment?.name && (
           <Detail
             label={i18n._(t`Execution Environment`)}
-            value={summary_fields.default_environment.name}
+            value={
+              <Link
+                to={`/execution_environments/${summary_fields.default_environment.id}/details`}
+              >
+                {summary_fields.default_environment.name}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Project/ProjectDetail/ProjectDetail.jsx:139]:
```suggestion
                {summary_fields?.default_environment?.name}
```

File: awx/ui_next/src/screens/Project/ProjectDetail/ProjectDetail.jsx [hunk 7]
```diff
@@ -132,10 +132,15 @@ function ProjectDetail({ project, i18n }) {
         {summary_fields?.default_environment?.name && (
           <Detail
             label={i18n._(t`Execution Environment`)}
-            value={summary_fields.default_environment.name}
+            value={
+              <Link
+                to={`/execution_environments/${summary_fields.default_environment.id}/details`}
```
Comment (jakemcdermott) [inline awx/ui_next/src/screens/Project/ProjectDetail/ProjectDetail.jsx:137]:
```suggestion
                to={`/execution_environments/${summary_fields?.default_environment?.id}/details`}
```

**Review by jakemcdermott — CHANGES_REQUESTED** [review]

File: awx/ui_next/src/screens/Organization/OrganizationDetail/OrganizationDetail.jsx (hunk 1 above)
Comment (nixocio) [reply awx/ui_next/src/screens/Organization/OrganizationDetail/OrganizationDetail.jsx:102]: I will update as per your suggestions. I was relying on the first if 

```
{summary_fields?.default_environment?.name ...

``` 
If that is the true, then we can proceed. Thanks!

Comment (jakemcdermott) [reply awx/ui_next/src/screens/Organization/OrganizationDetail/OrganizationDetail.jsx:102]: I see it now! yep :+1:

**Review by jakemcdermott — APPROVED** [review]

**Comment by jakemcdermott** [comment]
recheck

**Pull request merged**